easybuild/easyconfigs/m/MixClone/MixClone-1.1.6.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PythonBundle'
+
+name = 'MixClone'
+version = '1.1.6'
+
+homepage = 'https://github.com/uci-cbcl/MixClone#usage'
+description = """MixClone is a comprehensive software package to study the subclonal 
+structures of tumor genomes, including subclonal cellular prevalences estimation, allelic 
+configuration estimation, absolute copy number estimation and a few visualization 
+tools."""
+
+toolchain = SYSTEM  # or = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '2.7.16-GCCcore-8.3.0'),
+    ('SciPy-bundle', '2019.10-foss-2019b-Python-2.7.16'),
+]
+
+use_pip = True
+
+# All additional Python packages that are required, our package must be last in list 
+exts_list = [
+    ('pysam', '0.10.0', {
+        'checksums': ['1e925d2c9bf7b1753392da4fee3c0c184f8b062acd074905caa20569f87354e0'],
+    }),
+    (name, version, {
+        'source_urls': ['https://github.com/uci-cbcl/MixClone/archive/'],
+        'sources': ['%(version)s.tar.gz'],
+        'checksums': ['9284b0f6b133c904992ee840ca8d92ca47dde55448902add604c19ebf1aa8d8a'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/mixclone',
+             'lib/python%(pyshortver)s/site-packages/pysam',
+             ],
+}
+
+#sanity_check_commands = ["python -c 'import tensorflow; import tensorflow_probability'"]
+sanity_check_commands = ["MixClone.py -h"]
```

Comment:
That also means there should be a `bin/MixClone.py`, so include that in `sanity_check_paths`?